Doc/library/pprint.rst

```diff
@@ -217,9 +217,10 @@ let's fetch information about a project from `PyPI <https://pypi.org>`_::
    >>> import json
    >>> import pprint
    >>> from urllib.request import urlopen
-   >>> with urlopen('http://pypi.org/project/Twisted/json') as url:
+   >>> with urlopen('https://pypi.org/pypi/Twisted/json') as url:
    ...     http_info = url.info()
-   ...     raw_data = url.read().decode(http_info.get_content_charset())
+   ...     charset = http_info.get_content_charset(failobj="utf-8")
+   ...     raw_data = url.read().decode(charset)
```

Comment:
This is example of `pprint`.  So this part should be simple as possible:

```py
with urlopen('https://pypi.org/pypi/Twisted/json') as resp:
    project_info = json.load(resp)
```

Note that `json.load` and `json.loads` now support loading bytes encoded with UTF-8.
So no need to manual decode.